### DISCUSSION TITLE
Potential Issue with Release v11.1.3

### DISCUSSION THREAD

**Discussion by seve** [vercel/next.js#33747] [Help] [answered]
Is it possible that the v11.1.3 release was improperly cut? 

Looking at the latest canary tag for the release, there are a lot more commits on canary than actually made it to the patch.  I[ commented on a PR that I expected/wanted to see in the release](https://github.com/vercel/next.js/pull/28710#issuecomment-1021665525), but wasn't sure if it was going to get any eyes over there.

**Comment by tihuan** [comment]
+1 ! Especially the support for `node-sass` 6 seems to be missing from v11.1.3?

**Comment by balazsorban44** [comment, marked as answer]
Please see https://github.com/vercel/next.js/issues/34139#issuecomment-1034290706
